src/error_codes.jl

```diff
@@ -308,12 +310,10 @@
     EUNOWN,
 )
 
-
```

Comment:
Can I turn off this formatting for this file? Is there some annotation I can use to ignore a rule?

This spacing is organizational.

Reply:
yeah, we can instead run the formatter with `remove_extra_newlines=false,`

Reply:
But I don't want to turn off that rule for the whole package, just for this file.

Reply:
i see. i'm sure that's possible, but it might be the case of running with different settings on this file (e.g. if formatting were done via CI, or via a script vendored with the repo) ...not sure

Reply:
one option would be to put this file in a subdirectory with it's own `.JuliaFormatter.toml` config file https://domluna.github.io/JuliaFormatter.jl/dev/config/#Search-Rule-1

Reply:
although i don't know if you can use `.JuliaFormatter.toml` to turn off formatting for a directory...

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/302